cawg_identity/src/validator.rs

```diff
@@ -37,7 +37,7 @@ impl AsyncPostValidator for CawgValidator {
         partial_claim: &PartialClaim,
         tracker: &mut StatusTracker,
     ) -> c2pa::Result<Option<Value>> {
-        if label == "cawg.identity" {
+        if label.starts_with("cawg.identity") {
```

Comment:
This is generally the better practice now that assertions are correctly labeled with instances.